roles/addons/report/templates/graph-tool.py.j2

```diff
@@ -15,13 +15,13 @@ edge_color = g.new_edge_property("vector<float>")
 edge_text = g.new_edge_property("string")
 
 {% for network in networks %}
-{% if network not in report_settings.graph_tool_settings.excluded_networks %} 
+{#{% if network not in report_graph_tool_settings.excluded_networks %} #}
```

Comment:
why the comment? was it a test?

Reply:
It was working in the first version and I want to keep a track of this as it lay be relevant later. But for now, I do not use it.

Reply:
Well, it is tracked in git history 😁 . If you want to keep it as comment in the code, should you add a [TODO or FIXME tag](https://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags)?